src/main/java/com/targomo/client/api/request/RatingRequest.java

```diff
@@ -23,29 +23,40 @@ public class RatingRequest {
     private final String serviceUrl;
     private final String apiKey;
 
-    private final boolean showDetails;
+    private final Boolean showDetails;
+    private final Boolean forceRecalculate;
+    private final Boolean cacheResult;
```

Comment:
should we have a quick comment on each where we say what is the defautl?

Reply:
done!